### DISCUSSION TITLE
It won't let me put blocks

### DISCUSSION THREAD

**Discussion by Elabrax** [ScreamingSandals/BedWars#511] [Q&A]
When I start the game everything goes well until I have to place blocks, it eliminates them immediately and I do not have any region selected and the map is completely alone as I could solve it, the post does not give me any errors

**Comment by Misat11** [comment]
Does the arena contain any technical blocks like light source blocks in areas where the players are supposed to build? Is the arena covered by snow layer or similar block?

**Reply by Elabrax** [reply]
Will I send you photos?

**Reply by Misat11** [reply]
Photos of the map does not tell us much, but at least we can say it's not caused by snow. Still you have not answered to my question about light blocks (invisible blocks in 1.17 and newer emitting light similar to barrier but acts like air for normal players) which is usually a problem when a map is downloaded from somewhere. 

If players cannot build during the bedwars game, can they build while being in the arena, but not playing bedwars? Does it behave differently for OP? Are you sure there's no plugin that "protects" the arena?

**Reply by Elabrax** [reply]
There are barriers on the map, maybe that's it, but I, who am an op, can't build either, and I have worldguard, but I'm not using it in that world.

**Reply by Misat11** [reply]
Barrier and light source blocks are only interactable in creative mode.

**Reply by Elabrax** [reply]
I removed the blocks and it still doesn't let you put blocks and I recreated the arena again, by the way I'm using mohist 1.20.1 507 if you want I'll give you the mods and the server so you can see the problem

**Reply by boiscljo** [reply]
Mohist, Arclight and other hybrid server just totally break the plugin placement. Just don't use a hybrid server it won't work

**Comment by Elabrax** [comment]
![2023-11-14_02 26 08](https://github.com/ScreamingSandals/BedWars/assets/32414207/cc01a101-50a9-41fc-9c68-f716bc5db28d)
![2023-11-14_02 26 18](https://github.com/ScreamingSandals/BedWars/assets/32414207/1b2f1eb8-9619-40d8-b84e-f7ef87fa8033)
![2023-11-14_02 26 26](https://github.com/ScreamingSandals/BedWars/assets/32414207/0ae1278f-f561-48be-9338-d8cd091ff60a)
![2023-11-14_02 26 42](https://github.com/ScreamingSandals/BedWars/assets/32414207/271feab5-fd1b-45b9-8ef6-8f8939cf4c76)
![2023-11-14_02 25 30](https://github.com/ScreamingSandals/BedWars/assets/32414207/46b3f90a-85cd-4794-8a39-7bf83970d929)
![2023-11-14_02 25 41](https://github.com/ScreamingSandals/BedWars/assets/32414207/0560bcc2-85f9-430b-8db5-406194feb3fd)
![2023-11-14_02 25 50](https://github.com/ScreamingSandals/BedWars/assets/32414207/01aa485c-29ac-45fe-9215-4c108e11d5c8)
